pkg/cmd/mysqlout-to-csv/main.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/cockroachdb/cockroach/pkg/util"
+)
+
+var lineSep = flag.String("lines-terminated-by", "\n", "line separator character")
+var fieldsSep = flag.String("fields-terminated-by", "\t", "field separator character")
+var encloseChar = flag.String("fields-enclosed-by", "", "field enclosing character")
+var escapeChar = flag.String("fields-escaped-by", "\\", "escape character")
+
+var chunkCSVRows = flag.Int("chunk-rows", 0, "chunk csv into N rows per file (0 no chunking)")
+
+var progressFreq = flag.Int("progress", 1000, "number of rows per progress update (0 for no updates)")
+
+var bufSizeKb = flag.Int("buf-size-kb", 1024, "buffer size")
```

Comment:
does this really need to be an option?

Reply:
I guess not, I was experimenting with it to see if 1kb vs 1mb vs 100mb made a meaningful difference in time to convert our 180mb test case. It didn't on a MBP SSD, so I'll switch it to 64kb.